dbms/src/Storages/Transaction/KVStore.cpp

```diff
@@ -445,21 +442,13 @@ EngineStoreApplyRes KVStore::handleAdminRaftCmd(raft_cmdpb::AdminRequest && requ
 
         // After region split / merge, try to flush it
         const auto try_to_flush_region = [&tmt](const RegionPtr & region) {
-            if (tmt.isBgFlushDisabled())
+            try
             {
-                try
-                {
-                    tmt.getRegionTable().tryFlushRegion(region, false);
-                }
-                catch (const Exception & e)
-                {
-                    tryLogCurrentException(__PRETTY_FUNCTION__);
-                }
+                tmt.getRegionTable().tryFlushRegion(region, false);
             }
-            else
+            catch (const Exception & e)
```

Comment:
how about
`catch (...)`?

Reply:
Hi, thanks for reivew. but I don't understand what you mean, can you explain it in detail? Thanks a lot.